source/sundials/kinsol.cc

```diff
@@ -36,14 +36,10 @@
 
 #    include <deal.II/sundials/copy.h>
 
+#    include <kinsol/kinsol_direct.h>
 #    include <sundials/sundials_config.h>
-#    if DEAL_II_SUNDIALS_VERSION_GTE(3, 0, 0)
-#      include <kinsol/kinsol_direct.h>
-#      include <sunlinsol/sunlinsol_dense.h>
-#      include <sunmatrix/sunmatrix_dense.h>
-#    else
-#      include <kinsol/kinsol_dense.h>
-#    endif
+#    include <sunlinsol/sunlinsol_dense.h>
+#    include <sunmatrix/sunmatrix_dense.h>
```

Comment:
@bangerth I am puzzled by the compilation failure. The only difference I can spot is that you changed the order of include files.

Reply:
And I didn't even -- clang-format did that for me. Let me see what happens if I change that back...